playground/src/components/state-machines-details/state-machines-details.js

```diff
@@ -15,84 +15,116 @@ import {
 } from '@commercetools-frontend/ui-kit';
 import { fetchStateMachine } from './actions';
 
-export class StateMachinesDetails extends React.Component {
-  static displayName = 'StateMachinesDetails';
+const initialState = {
+  isLoading: true,
+  error: null,
+  data: null,
+};
+
+// NOTE: using `useEffect` for asynchronously update the state, thus re-rendering
+// the component, has currently a limitation when testing the component, as it throws
+// a warning about `act`. This will be fixed in react@16.9.
+// In the meantime we can use a plain old stateful class component.
+class StateMachinesDetailsFetcher extends React.Component {
```

Comment:
Why did this compnonent change at all?

Reply:
Yeah so actually I wanted to use only hook (with `useEffect`) but there is still this issue with `act` in testing. So I just extracted the fetching logic into a stateful component, but it's just a temporary thing. Eventually it should all be hooks.

Reply:
Yeah I know this problem. Haf it with sign up components too.